app/models/partner.rb

```diff
@@ -211,6 +211,26 @@ def export_json
     }
   end
 
+  def families_served
+    families.count
+  end
+
+  def children_served
+    children.count
+  end
+
+  def family_zipcodes
+    families.pluck(:guardian_zip_code).uniq.count
+  end
```

Comment:
Could we put these under private? I generally try to keep the public interface of classes as small as possible. I find that to keep classes small in size and also makes us less liable having to test them :).